### ISSUE TITLE
Can you give me some guidance ？

### ISSUE THREAD

**Issue by LDZ233-ldz** [google-research/simclr#120]
Thank you for your code,
Can it be used for training on labeled COCO datasets
Can you give me some guidance on how to use simclr for downstream tasks, such as supervised target detection?

**Comment by chentingpc** [comment]
The method presented here is used for pretraining of ResNet backbone, which can then be transferred fro classification, detection, segmentation, etc. So we don't use labels in pretraining (whether it's bounding box or class label), and you could use the labels in fine-tuning stage (after pretraining).
